cmake/ConfigDefault.cmake

```diff
@@ -39,7 +39,7 @@ set (GMT_VERSION_YEAR "2019")
 
 # The GMT package version.
 set (GMT_PACKAGE_VERSION_MAJOR 6)
-set (GMT_PACKAGE_VERSION_MINOR 0)
+set (GMT_PACKAGE_VERSION_MINOR 1)
```

Comment:
This should not be here. The other time was to merge in master. This one goes to 6.0